WORKSPACE

```diff
@@ -2067,9 +2061,15 @@ rpm(
 )
 
 rpm(
-    name = "glibc-langpack-ast-0__2.28-164.el8.aarch64",
-    sha256 = "9a419078253d9907e2f4fe886754157c1b9b6697d43f5f14c1a21440610cbcb0",
-    urls = ["http://mirror.centos.org/centos/8-stream/BaseOS/aarch64/os/Packages/glibc-langpack-ast-2.28-164.el8.aarch64.rpm"],
+    name = "glibc-langpack-bho-0__2.28-164.el8.aarch64",
+    sha256 = "32100a52177b16ac6e2e8d840136afa2bb40682903835862899942f8faa64858",
+    urls = ["http://mirror.centos.org/centos/8-stream/BaseOS/aarch64/os/Packages/glibc-langpack-bho-2.28-164.el8.aarch64.rpm"],
+)
+
+rpm(
+    name = "glibc-langpack-ca-0__2.28-164.el8.x86_64",
+    sha256 = "9e86cfb0afd8d7b39521acd4742890bc9bdbf7b3825bad33981db6844a1c517e",
+    urls = ["http://mirror.centos.org/centos/8-stream/BaseOS/x86_64/os/Packages/glibc-langpack-ca-2.28-164.el8.x86_64.rpm"],
```

Comment:
This and other changes to `WORKSPACE` as well as `rpm/BUILD.bazel`, except for those specifically about xattrs of course, are unrelated to the commit at hand and should live somewhere else.

Reply:
This also went to the final `make rpm-deps` run in the last commit.